### ISSUE TITLE
Cannot find module 'request'

### ISSUE THREAD

**Issue by Gmulti** [image-charts/javascript#9]
Hi,

I just installed the library to start using your tool.
I use NextJS (latest version) and after a first test, here is what I get when I try to generate a URL.

It is noted that it can be used on the Node side, it also seems strange to me that if a dependency is missing, it is not installed?

<img width="322" alt="Capture d’écran 2022-04-29 à 15 22 57" src="https://user-images.githubusercontent.com/5682535/165952860-95fc3c2c-cf0e-4881-befb-15fe636b329e.png">

Example : 
```js
const pie = ImageCharts().cht('lc').chd('t:50,20,30,43,2,3,456,7').chxl('1:|01/04|02/04|03/04|04/04|04/04|04/04|04/04|04/04').chtt('New Visitors').chm('B,76A4FB,0,0,0')

pie.toURL()
```

An idea ?

**Comment by xsstomy** [comment]
same error, cannot find module  'request'

mac os  m1 pro 14



```
   const chart_path = './chart.png';

    ImageCharts()
        .cht('bvg') // vertical bar chart
        .chs('300x300') // 300px x 300px
        .chd('a:60,40') // 2 data points: 60 and 40
        .toFile(chart_path)
        .then(() => console.log('Image chart written at %s', chart_path))
```

**Comment by FGRibreau** [comment]
Thanks for your report, fixed in v6.0.19 :+1:
